src/pages/self-hosted.tsx

```diff
@@ -67,10 +66,10 @@ const SelfHostedPage: React.SFC<{}> = () => (
             {/* ----- Section Customizations ----- */}
 
             <ActionCard
-                title="Need Any Customizations?"
+                title="Need Help Or Have Questions?"
                 text={
                     <React.Fragment>
-                        <p>We’re happy to adjust Gitpod to your needs.<br />Please get in touch for details.</p>
+                        <p>Please reach out.<br />We’re happy to help getting <br />the perfect Gitpod setup for your team.</p>
```

Comment:
This looks a bit like the Star Wars intro text. 😂

<img width="490" alt="Screenshot 2020-05-06 at 11 00 11" src="https://user-images.githubusercontent.com/599268/81159268-15f6aa80-8f89-11ea-921a-c120f904d24b.png">

Maybe like so?

<img width="484" alt="Screenshot 2020-05-06 at 11 01 55" src="https://user-images.githubusercontent.com/599268/81159314-21e26c80-8f89-11ea-9346-fbe6c8e548cb.png">

```suggestion
                        <p>Please reach out. We’re happy to help build<br />the perfect Gitpod setup for your team.</p>
```